lib/git-helpers.js

```diff
@@ -31,7 +31,7 @@ module.exports = {
     // console.log(`git config --replace-all remote.origin.fetch +refs/heads/*:refs/remotes/origin/*`)
     exec(`git config --replace-all remote.origin.fetch +refs/heads/*:refs/remotes/origin/*`)
     // console.log(`git fetch`)
-    exec(`git fetch`)
+    exec(`git fetch gk-origin`)
```

Comment:
It would be ideal to have this `gk-origin` value stored isomewhere, as it is referenced here and in `upload.js` but not sure where to put it.